02-technical-guides/currency-conversion-with-lookup/main.ipynb

```diff
@@ -353,15 +353,15 @@
     {
```

Comment:
<p>Dates table has a weird name? Expected?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/atoti/atoti/pull/883/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>This seems to be something to do with <code>create_parameter_hierarchy_from_members()</code>. This is negligible at the moment so I won't block the PR on this, but will bubble up the issue to the corresponding stakeholders.</p>